VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
Besides `AppPackages/` folder which contains .appxsym, .appxbundle, .cer, .ps1 and many other file types, and the project root folder .appx files, I didn't see any other output files to be excluded in my Git repo. I would keep the `AppPackages/` folder just in case someone creates the Store packages there.